bot.py

```diff
@@ -70,10 +75,17 @@
 pbCooldown = 100
 polling_list = dict()
 
+#Initialization
+print 'Starting boot sequence'
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
```

Comment:
This works for your local instance, but should connect to the pbot db, look into passing the connection string and user creds as env variables